feature/experimental/isis/ate_tests/isis_change_lsp_lifetime_test/metadata.textproto

```diff
@@ -39,5 +37,6 @@ platform_exceptions: {
     missing_value_for_defaults: true
     interface_enabled: true
     default_network_instance: "default"
+    isis_instance_enabled_required: true          
```

Comment:
Why the flip? If isis.Global().Instance leaf is not supported, I would name the deviation as `isis_global_instance_name_unsupported`

Reply:
Its changes as per this discussion here https://github.com/openconfig/featureprofiles/pull/1936#discussion_r1272656562.
This leaf is deprecated , but some other vendors still require it i think, so have named this deviation such.

Reply:
Aah. Thanks for reminding me! LGTM!